src/planscape/forsys/views.py

```diff
@@ -214,7 +216,9 @@ def run_forsys_generate_project_areas_for_a_single_scenario(
         headers.FORSYS_TREATMENT_ELIGIBILITY_HEADER,
         "" if output_scenario_name is None else output_scenario_name,
         "" if output_scenario_tag is None else output_scenario_tag,
-        enable_kmeans_clustering)
+        enable_kmeans_clustering,
+        "" if max_cost_per_project_in_usd is None else max_cost_per_project_in_usd,
```

Comment:
instead of passing a string if max_cost_per_project_in_usd is None, is there another value that could be passed that'd make the types more consistent and/or the logic simpler in the R script?

A stray thought is: if we have error-checking in forsys_request_params.py that ensures only one of the two constraints is None, then it'd be ok for the one that's None to be planning area acreage or the cost to treat the entire planning area.

Wdyt?

Reply:
I was thinking we could pass in a 0 or -1, but that still seems weird.  The py to R handoff can't handle None (from py) which is why lines 217 and 218 follow the same pattern.

I'd like to defer changing this pattern for later; we're using "" from the python side as "no value", and R requires something on its end.  I also am not quite ready to put in default forsys behavior (e.g. treat no cost limit provided as $100B) into python quite yet.

I did realize that there were two comment lines in the R that no longer made sense and perhaps eluded a sense of confusion there.

Reply:
sg! I also wouldn't want to hardcode default forsys behavior for None.